Libraries/Components/Touchable/TouchableNativeFeedback.android.js

```diff
@@ -185,7 +185,10 @@ var TouchableNativeFeedback = React.createClass({
 
   _handleResponderMove: function(e) {
     this.touchableHandleResponderMove(e);
-    this._dispatchHotspotUpdate(e.nativeEvent.pageX, e.nativeEvent.pageY);
+    var [elementWidth, elementHeight] = (this.pressInLocation)
```

Comment:
Not sure the trick with declaring two variables is worth it. Might be more readable to compute each separately: the line below is very long and it's not immediately clear what `[0, 0]` does. Also, are element width and height actually 0 if there is no `pressInLocation`? Should the variables be named differently?